pytorch/inference/docker/1.10/py3/Dockerfile.graviton

```diff
@@ -64,6 +64,12 @@ RUN /opt/conda/bin/conda install -y \
 
 RUN /opt/conda/bin/conda clean -ya
 
+# Conda installs links for libtinfo.so.6 and libtinfo.so.6.2 both
```

Comment:
Can we also put the conda issue link in the comment?

Reply:
also added the issue link in pytorch-1.9-trainning dockerfile